third_party/service-catalog/manifests/catalog/templates/apiserver-deployment.yaml

```diff
@@ -135,4 +135,5 @@ spec:
           - key: tls.key
             path: apiserver.key
       - name: etcd-data-dir
-        emptyDir: {}
+        persistentVolumeClaim:
+          claimName: release-name-catalog-etcd
```

Comment:
Can we pick a better name? I think we can choose whatever we want as long as the two files match.
maybe service-catalog-persistent-etcd

Reply:
Looking more at these files, I see this is a pattern we have throughout. Maybe another reviewer would have more context: maybe we should worry about these changes in a second PR.

Reply:
Right now all the names were automatically generated using the chart template. If we want to rename it, I think we would have to modify .Release.Name in the chart template helper, which would change several other names as well. I agree we could put the changes in a separate PR

Reply:
I'm going to resolve these comments, since I opened an issue to address these at a later date: https://github.com/google/kf/issues/515